client/client.go

```diff
@@ -0,0 +1,64 @@
+package client
```

Comment:
Sounds sensible although maybe the following is more idiomatic?
```
client/proxy/proxy.go         // package proxy
client/proxy/proxy_test.go  
client/mixer/mixer.go         // package mixer
client/mixer/mixer_test.go  
```
Single package name per directory?